tests/fixtures/apps/tb/case/case_qc_metrics.yaml

```diff
@@ -15,18 +15,18 @@ recipe:
     version: 1.5.0
   pedigree_check:
     sample_order:
-      - father
```

Comment:
why do we change the samples used?

Reply:
So that they correspond to the other fixture files, otherwise it will be inconsistencies when using information from different sources